uitools/toolkitwidgets/src/Esri/ArcGISRuntime/Toolkit/NorthArrow.cpp

```diff
@@ -49,13 +51,15 @@ NorthArrow::NorthArrow(QWidget* parent) :
   QLabel(parent),
   m_controller(new NorthArrowController(this))
 {
+  setScaledContents(true);
```

Comment:
In my tests, I was surprised the provided code doesn't display the NorthArrow correctly. I needed to add `setScaledContents` and `resize` to fix that.

Reply:
Good find! Not sure at what point it stopped working.